src/main.rs

```diff
@@ -173,20 +188,31 @@ fn main() {
                      sock=<socket_path>\"",
                 )
                 .takes_value(true)
-                .min_values(1),
+                .min_values(1)
+                .group("vm-config"),
         )
         .arg(
             Arg::with_name("v")
                 .short("v")
                 .multiple(true)
-                .help("Sets the level of debugging output"),
+                .help("Sets the level of debugging output")
+                .group("vmm-config"),
         )
         .arg(
             Arg::with_name("log-file")
                 .long("log-file")
                 .help("Log file. Standard error is used if not specified")
                 .takes_value(true)
-                .min_values(1),
+                .min_values(1)
+                .group("vmm-config"),
+        )
+        .arg(
+            Arg::with_name("api-socket")
+                .long("api-socket")
+                .help("HTTP API socket path (UNIX domain socket).")
+                .takes_value(true)
```

Comment:
Having to give a socket path sucks.  We should default to `/run/user/<uid>/cloud-hypervisor.sock`

Reply:
Agreed, will fix.

Reply:
Maybe add `.<pid>`  at the end.